### DISCUSSION TITLE
Copy-on-select doesn't work in Konsole?

### DISCUSSION THREAD

**Discussion by J-Kappes** [zellij-org/zellij#2605] [Q&A] [answered]
It doesn't work for me in Konsole but it does in Alacritty. Can anybody reproduce this? Not that I expect either side to fix this issue if it's real.
```
Konsole 22.12.3
Zellij 0.37.2
```

**Comment by tlinford** [comment, marked as answer]
konsole doesn't support osc52 yet: https://bugs.kde.org/show_bug.cgi?id=372116

please have a lot at the [docs](https://zellij.dev/documentation/faq.html#copy--paste-isnt-working-how-can-i-fix-this) for some possible alternatives
